index.bs

```diff
@@ -1687,6 +1689,18 @@ ArrayBufferRemoteValue = {
   ?internalId: InternalId,
 }
 
+NodeListRemoteValue = {
```

Comment:
Is it intentional that `NodeListRemoteValue` and HTMLCollectionRemoteValue` omit the `type` property? How should a local end detect these constructs when encountered?

Reply:
Haha, no that's just incompetence. I'll fix.